{"url":"https://api.github.com/repos/owncloud/core/issues/493","labels_url":"https://api.github.com/repos/owncloud/core/issues/493/labels{/name}","comments_url":"https://api.github.com/repos/owncloud/core/issues/493/comments","events_url":"https://api.github.com/repos/owncloud/core/issues/493/events","html_url":"https://github.com/owncloud/core/issues/493","id":8431572,"number":493,"title":"invalid quotes within node.php in connection with postgresql","user":{"login":"b5r1oj0a9g","id":1625487,"avatar_url":"https://gravatar.com/avatar/4a509fd00a19412ac02bb9bbe4c1eeec?d=https%3a%2f%2fidenticons.github.com%2fa7ea4747c59a8b0e3c9e5ee3d737e566.png&r=x","gravatar_id":"4a509fd00a19412ac02bb9bbe4c1eeec","url":"https://api.github.com/users/b5r1oj0a9g","html_url":"https://github.com/b5r1oj0a9g","followers_url":"https://api.github.com/users/b5r1oj0a9g/followers","following_url":"https://api.github.com/users/b5r1oj0a9g/following{/other_user}","gists_url":"https://api.github.com/users/b5r1oj0a9g/gists{/gist_id}","starred_url":"https://api.github.com/users/b5r1oj0a9g/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/b5r1oj0a9g/subscriptions","organizations_url":"https://api.github.com/users/b5r1oj0a9g/orgs","repos_url":"https://api.github.com/users/b5r1oj0a9g/repos","events_url":"https://api.github.com/users/b5r1oj0a9g/events{/privacy}","received_events_url":"https://api.github.com/users/b5r1oj0a9g/received_events","type":"user","site_admin":false},"labels":[{"url":"https://api.github.com/repos/owncloud/core/labels/duplicate","name":"duplicate","color":"cccccc"}],"state":"closed","assignee":null,"milestone":null,"comments":4,"created_at":"2012-11-16t20:53:42z","updated_at":"2012-11-19t13:04:12z","closed_at":"2012-11-19t13:04:12z","pull_request":{"html_url":null,"diff_url":null,"patch_url":null},"body":"accessing a shared folder via the web ui results in an empty page and the following error message occured:\n```\n[fri nov 16 21:07:43 2012] [error] [client 192.0.43.10] php fatal error:  uncaught exception 'pdoexception' with message 'sqlstate[42703]: undefined column: 7 error:  column \"/shared\" does not exist\\nline 1: delete from \"properties\" where \"propertypath\" = \"/shared\"\\n\n```\n\nbecause postgresql doesn't allow allow that kind of quotation i had to made the following changes to get owncloud working again:\n```\n*** lib/connector/sabre/node.php.orig\t2012-11-14 09:32:11.000000000 +0100\n--- lib/connector/sabre/node.php\t2012-11-16 21:09:21.132447037 +0100\n***************\n*** 279,283 ****\n  \t\t//remove etag for all shared folders\n  \t\t$query = oc_db::prepare( 'delete from `*prefix*properties`'\n! \t\t\t\t.' where `propertypath` = \"/shared\"'\n  \t\t);\n  \t\t$query->execute(array());\n--- 279,283 ----\n  \t\t//remove etag for all shared folders\n  \t\t$query = oc_db::prepare( 'delete from `*prefix*properties`'\n! \t\t\t\t.' where `propertypath` = \\'/shared\\''\n  \t\t);\n  \t\t$query->execute(array());\n```\n\nenvironment:\ncentos 6\nowncloud 4.5.2\npostgresql 9.1","closed_by":{"login":"karlitschek","id":867445,"avatar_url":"https://gravatar.com/avatar/87ce2b4531ee1a0c32b50e0d8f049224?d=https%3a%2f%2fidenticons.github.com%2f06e7098636caebfc9c30c9f52eb905df.png&r=x","gravatar_id":"87ce2b4531ee1a0c32b50e0d8f049224","url":"https://api.github.com/users/karlitschek","html_url":"https://github.com/karlitschek","followers_url":"https://api.github.com/users/karlitschek/followers","following_url":"https://api.github.com/users/karlitschek/following{/other_user}","gists_url":"https://api.github.com/users/karlitschek/gists{/gist_id}","starred_url":"https://api.github.com/users/karlitschek/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/karlitschek/subscriptions","organizations_url":"https://api.github.com/users/karlitschek/orgs","repos_url":"https://api.github.com/users/karlitschek/repos","events_url":"https://api.github.com/users/karlitschek/events{/privacy}","received_events_url":"https://api.github.com/users/karlitschek/received_events","type":"user","site_admin":false}}